website/docs/r/service_principal_entitlement.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_service_principal_entitlement"
+description: |-
+  Manages a service principal entitlement within Azure DevOps organization.
+---
+
+# azuredevops_service_principal_entitlement
+
+Manages a service principal entitlement within Azure DevOps.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_service_principal_entitlement" "example" {
+  origin_id = "90c9f86a-8ffb-4ff4-bae5-729100074a0c"
+}
+```
+
+## Argument Reference
+
+- `origin_id` - (Required) The object ID of the enterprise application.
+- `origin` - (Optional) The type of source provider for the origin identifier. Defaults to `aad`.
```

Comment:
any other possible values ​​besides the default value?

Reply:
Not sure. The [docs](https://learn.microsoft.com/en-us/rest/api/azure/devops/memberentitlementmanagement/service-principal-entitlements/add?view=azure-devops-rest-7.1&tabs=HTTP) say AAD, AD, MSA, but I'm not sure if that list is exhaustive

Reply:
we can add `AD`, `AAD`, `MSA` to the doc